tests/integration/defs/agg_unit_mem_df.csv

```diff
@@ -113,5 +113,4 @@ unittest/_torch/thop/parallel,NVIDIA B200,16,
 "unittest/_torch/auto_deploy/unit/singlegpu -k ""not test_trtllm_bench_backend_comparison""",NVIDIA B200,4,
 unittest/_torch/attention,NVIDIA H100,4,
 unittest/_torch/misc,NVIDIA H100,4,
-unittest/_torch/speculative,NVIDIA H100,2,
```

Comment:
Do we also need to disable it for B200?

Reply:
We only observed intermittent timeout issues in H100. So, let's reduce pytest workers in H100 first.